### DISCUSSION TITLE
rabbitmq-docker Facing issue LDAP access denied: user 'xxxx' - Not management user

### DISCUSSION THREAD

**Discussion by raynent** [rabbitmq/rabbitmq-server#10554] [Questions] [answered]
docker compose file 
![image](https://github.com/docker-library/rabbitmq/assets/143066456/22d10caf-5440-4f1f-b3e4-6f85fa2cf688)

LDAP config had set ldap as auth_backends
![image](https://github.com/docker-library/rabbitmq/assets/143066456/f710e13e-980a-4910-9e23-eeee5a9fddb0)

ERROR showing when login with ldap account
![image](https://github.com/docker-library/rabbitmq/assets/143066456/f3dfb0d5-5330-4c99-adf2-40043d576164)
![image](https://github.com/docker-library/rabbitmq/assets/143066456/8533e34f-d612-42ed-a8a4-c6c7aaddfb63)

Tried add something into advance.config, but seem it doesn't load advance.config file when start container , so i guess the advance.config way doesn't support on docker version?
![image](https://github.com/rabbitmq/rabbitmq-server/assets/143066456/ead9c2ba-f8a0-4edd-9220-280c54832763)

Any idea ? thanks a lot

**Comment by lukebakken** [comment]
@raynent it is **EXTREMELY IMPOLITE** to directly ping anyone who provides free software to you.

Secondly, screenshots are not a good way to provide code or configuration. You must share an archive or, even better, a git repository with everything I need.

Finally, the root of this problem is that you are not making `advanced.config` available within your Docker container. This is Docker 101. Create your image based on the official RabbitMQ image and `COPY` it in, or mount it as a volume.

**Comment by raynent** [comment, marked as answer]
solved , the solution is put all files in /etc/rabbitmq path instead if default path /etc/rabbitmq/conf.d
